### PR TITLE
Verify logLevel

### PR DESCRIPTION
**Pull request by TuHuynhVan**
## Proposed changes

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [ ] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: lib/utils/Logger.js [hunk 1]
```diff
@@ -20,10 +20,20 @@ class Logger {
...
+         */
+        if (options.logLevel != 'silent' && options.logLevel != 'command' && options.logLevel != 'result' && options.logLevel != 'error' && options.logLevel != 'verbose') {
+            throw new Error(`logLevel "${options.logLevel}" is not valid.`)
+        }
+
         this.logLevel = options.logLevel
 
         this.setupWriteStream(options)
 
+
```
Comment (christian-bromann) [inline lib/utils/Logger.js]: Is the extra line needed?

File: lib/utils/Logger.js [hunk 2]
```diff
@@ -20,10 +20,20 @@ class Logger {
          * error : error only
          * verbose : command + data + result
          */
+
+
+        /**
+         * logLevel verification
+         */
+        if (options.logLevel != 'silent' && options.logLevel != 'command' && options.logLevel != 'result' && options.logLevel != 'error' && options.logLevel != 'verbose') {
```
Comment (christian-bromann) [inline lib/utils/Logger.js]: I think it is easier to have all log options in an array and check if it includes the given logLevel

**Comment by TuHuynhVan** [comment]
@christian-bromann 
I fixed them all. Could you please help to review?
I also created https://github.com/webdriverio/webdriverio/pull/2426

**Comment by christian-bromann** [comment]
@TuHuynhVan can you please merge what you have in #2426 into this? No reason to have separated PRs

**Comment by TuHuynhVan** [comment]
@christian-bromann Yes, that is not good for separated PR. Done.

File: lib/helpers/constants.js [hunk 3]
```diff
@@ -1,4 +1,9 @@
 /**
+ * logLevel
+ */
+let LOG_LEVELS = ['silent', 'command', 'result', 'error', 'verbose'];
```
Comment (christian-bromann) [inline lib/helpers/constants.js]: Let's make this a `const`

File: lib/utils/Logger.js [hunk 4]
```diff
@@ -20,6 +20,15 @@ class Logger {
          * error : error only
          * verbose : command + data + result
          */
+
+
+        /**
+         * logLevel verification
+         */
+        if (LOG_LEVELS.indexOf(options.logLevel === -1)) {
```
Comment (christian-bromann) [inline lib/utils/Logger.js]: Let's use `LOG_LEVELS.includes(options.logLevel)`

**Comment by TuHuynhVan** [comment]
@christian-bromann Thank you so much. I fix them all. Please help to review.

**Pull request merged**